proto/api.proto

```diff
@@ -943,6 +943,8 @@ message RouteStep {
   double outAmount = 4;
   double outAmountMin = 5;
   StepProject project = 6;
+  // if poolAddress is defined, it determines the outToken
```

Comment:
It determines both in and out token right? Also, shouldn't we use the `project` field for this?

```
message StepProject {
  string label = 1;
  string id = 2;
}
```

Seems to me that we should use the ID field in there